package.json

```diff
@@ -30,15 +30,16 @@
     "adm-zip": "0.4.7",
     "gulp": "3.9.0",
     "gulp-util": "3.0.4",
+    "markdown-toc": "^1.2.0",
```

Comment:
?

Reply:
@ericsciple, I added this as a dev dependency so that you can easily generate a TOC for .md files.  Without having to _hand_ generate one. Usage looks something like this: .\node_modules\.bin\markdown-toc .\README.md